File: hw1/citytest.py
import unittest

from city import City

class CityTest(unittest.TestCase):
    def testDefCity(self):
        size : float = 0.86
        name : str = "Ouray"
        population : int = 1000

        city : City = City(size, name, population)
        self.assertEqual(city.population, population)
        self.assertEqual(city.name, name)
        self.assertEqual(city.size, size)

    def cityType(self):
        size : float = 0.86
        name : str = "Ouray"
        population : int = 1000

        city : City = City(size, name, population)
        self.assertEqual(city.density(), "Town")

if __name__ == '__main__':
    unittest.main()

File: hw3/test/citytest.js

const assert = require('assert');
const chai = require('chai');
const { isMainThread } = require('worker_threads');
const { City }  = require('../city');

const expect = chai.expect;
const assertEq = assert.deepStrictEqual

describe('City', function() {
    describe('#constructor', function() {
        it('can be made', function() {
            const size = 0.86;
            const name = "Ouray";
            const population = 1000;
            const city = new City(size, name, population);
            assertEq(city.population, population);
            assertEq(city.name, name);
            assertEq(city.size, size);
            assertEq(city.density(), "Town");
        });
    });
});